include/eve/module/core/regular/rat.hpp

```diff
@@ -56,25 +56,24 @@ struct rat_t : elementwise_callable<rat_t, Options>
 //!   @code
 //!   namespace eve
 //!   {
-//!      template<eve::floating_value T>
-//!      eve::zipped<T,T> rat(T x, T tol = T(1.0e-6)*eve::abs(x)) noexcept;
+//!      // Regular overloads
+//!      constexpr auto rat(floating_value auto x)                           noexcept;
+//!      constexpr auto rat(floating_value auto x, floating_value auto tol)  noexcept;
 //!   }
 //!   @endcode
 //!
 //!   **Parameters**
 //!
 //!     * `x` :  [argument](@ref eve::value).
-//!     * `tol`: [floating_value](@ref eve::value). By  default `tol`
-//!       is equal to `T(1.0e-6)*``eve::abs``(x)`.
+//!     * `tol`: [floating value](@ref eve::value). By  default `tol`
+//!        is equal to `1.0e-6*eve::abs(x)`.
 //!
 //!   **Return value**
 //!
-//!     A tuple of values with the same type as `x` containing the
-//!     [elementwise](@ref glossary_elementwise) numerator and denominator of the rational
-//!     number approximating `x`.
+//!     A tuple-like of values with the same type as `x` containing the
```

Comment:
x is not a tuple so the tuple-like value can't be of the same type.
Try
"A tuple-like value which elements are of the same type as `x`"